### PR TITLE
First attempt at a working setup script

### PR DESCRIPTION
**Pull request by a-willis**
A script that will build docker images, run db:create, and bring up the web server
This will be used by the development repository 

Test plan: run ./setup.sh. When done, visit platformweb:3020 in browser

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/zsh
+
+cp .env.example .env
+cp .env.database.example .env.database
+docker-compose build
+docker-compose up -d platformdb 
+docker-compose up platformweb
```
Comment (hcli-work) [inline setup.sh:7]: how does this even work? wouldn't it just hang here?

Comment (hcli-work) [reply setup.sh:7]: If this works and is really necessary, would you mind just putting in a small comment?

**Review by hcli-work — CHANGES_REQUESTED** [review]

File: setup.sh (hunk 1 above)
Comment (a-willis) [reply setup.sh:7]: The db:create command won't work without trying to run platformweb first. I'll add a comment to explain more

**Review by hcli-work — APPROVED** [review]
lgtm!

File: setup.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/zsh
+
+cp .env.example .env
+cp .env.database.example .env.database
+docker-compose build
+docker-compose up -d platformdb 
+docker-compose up platformweb #this is a hack. it will exit with an error but needed for db:create
+docker-compose run platformweb bundle exec rake db:create db:schema:load db:migrate db:seed
```
Comment (sadleb) [inline setup.sh:8]: The steps in the README don't work? E.g. running
```
docker-compose up -d
docker-compose run platformweb bundle exec rake db:create db:schema:load db:migrate db:seed
```
And then restarting.

**Pull request merged**